app/views/catalog/_arclight_document_header_collection.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class='card document-header'>
+  <div class='card-block row'>
+    <div class='col-9'>
+      <%= document.fetch('repository_ssm', []).first %>
+    </div>
+    <div class='col-3 text-right'>
+      <%= t(:'arclight.views.show.collection_id', id: document.fetch('unitid_ssm', []).first) %>
```

Comment:
Wondering if there's something better we can do that hard-coding these into our views.